pipelines/ceus/tasks/fcz_correlation.py

```diff
@@ -52,6 +52,9 @@ def _task(self):
 
         for base in self.fcz_names:
             for target in self.projection_locations:
+
+                if base == 'FCZ10': #FCZ10 shows unecpected behavior
```

Comment:
why not just explicitly remove `FCZ10` from the list instead - would be clearer.

Reply:
I'm going to change this to be in a config file.